### DISCUSSION TITLE
QuestPDF in Azure Functions

### DISCUSSION THREAD

**Discussion by douglott** [QuestPDF/QuestPDF#541] [Show and tell]
For what it is worth, I have QuestPDF working in Azure Functions.  We are using this as a web hook endpoint to deliver PDF's to other applications with one example being Salesforce.

**Comment by MarcinZiabek** [comment]
The library should work without any problems in Azure Functions. I am not aware of any difficulties 😁

**Reply by douglott** [reply]
It's fast!  ⚡️

**Comment by MarcinZiabek** [comment]
For future reference, starting from the QuestPDF `2024.3.X` release, the library does not require any additional dependencies/packages to successfully run on various operating systems. 

At the moment of writing this message, the following platforms are supported: 
- win-x64, 
- linux-x64, 
- linux-arm64, 
- linux-musl-x64 (Alpine), 
- osx-x64, 
- osx-arm64.

If you are experiencing any issues, please change the `Platform target` property to `x64` in your project settings. 

Also, you may need to ensure that your Azure Functions instance uses the x64 worker process by setting the `Use32BitWorkerProcess` flag to false.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
